openwisp_controller/subnet_division/templates/admin/input_filter.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+
+<h3>{% blocktrans with filter_title=title %} By {{ filter_title }} {% endblocktrans %}</h3>
```

Comment:
is the space before `By` on purpose?

Reply:
Just to keep code breezy. I checked through the web debugger tool,  all headings of filters have a leading space character. I think it is good to leave it here. 
![Screenshot from 2021-03-26 23-45-27](https://user-images.githubusercontent.com/32094433/112675594-5dca3d00-8e8d-11eb-8396-b71d558d19a5.png)